src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleSummary.js

```diff
@@ -41,12 +41,6 @@ const CSAwRuleSummary = ({ text, truncate, intl, ...props }) => {
             </TextContent>
         </StackItem>
     );
-
-};
-
-CSAwRuleSummary.defaultProps = {
-    truncate: true,
-    link: null
```

Comment:
How about the link prop? Is it safe not to give a default value?

Reply:
Yes, the link prop is not used anywhere directly, there are `...props` passed into `StackItem`, but it does not have such prop